packages/CopyInput/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@paprika/copy-input",
+  "version": "1.0.0",
```

Comment:
@nahumzs should this be set to a lower version so we can "promote" it when we publish?

Reply:
I think is fine it will get published with 1.0.0-alpha and when promoted will be 1.0.0 👍